tests/api/test_views.py

```diff
@@ -592,65 +592,76 @@ class TestEnterpriseCatalogAPIViews(APITest):
-                        'test-user'
-                    ]
-                },
-            ],
-            [
-                {
-                    'id': 1,
-                    'name': 'Enterprise Dummy Catalog',
-                    'query': '*',
-                    'courses_count': 22,
-                    'viewers': []
-                },
-                {
-                    'id': 2,
-                    'name': 'Enterprise All Biology',
-                    'query': 'title:*Biology*',
-                    'courses_count': 3,
-                    'viewers': []
-                },
-                {
-                    'id': 3,
-                    'name': 'Test User Catalog',
-                    'query': '*',
-                    'courses_count': 22,
-                    'viewers': [
-                        'test-user'
+            {
+                'count': 3,
+                'next': 'http://testserver/enterprise/api/v1/catalogs/?page=3',
+                'previous': 'http://testserver/enterprise/api/v1/catalogs/?page=1',
+                'results':
+                    [
+                        {
+                            'id': 1,
+                            'name': 'Enterprise Dummy Catalog',
+                            'query': '*',
+                            'courses_count': 22,
+                            'viewers': []
+                        },
+                        {
+                            'id': 2,
+                            'name': 'Enterprise All Biology',
+                            'query': 'title:*Biology*',
+                            'courses_count': 3,
+                            'viewers': []
+                        },
+                        {
+                            'id': 3,
+                            'name': 'Test User Catalog',
+                            'query': '*',
+                            'courses_count': 22,
+                            'viewers': [
+                                'test-user'
+                            ]
+                        },
                     ]
-                },
-            ],
+            },
+            {
+                'count': 3,
+                'next': 'http://testserver/enterprise/api/v1/catalogs/?page=3',
+                'previous': 'http://testserver/enterprise/api/v1/catalogs/?page=1',
+                'results':
+                [
+                    {
+                        'id': 1,
+                        'name': 'Enterprise Dummy Catalog',
+                        'query': '*',
+                        'courses_count': 22,
+                        'viewers': []
+                    },
+                    {
+                        'id': 2,
+                        'name': 'Enterprise All Biology',
+                        'query': 'title:*Biology*',
+                        'courses_count': 3,
+                        'viewers': []
+                    },
+                    {
+                        'id': 3,
```

Comment:
It seems you can further refine the test data by using only one record with id `2` as you have already added next and previous links.